### PR TITLE
Code review

### PR DESCRIPTION
**Pull request by danielamoreno699**
#Comments
-Add linters.yml

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        <form id="form" action="https://www.freecodecamp.com/email-submit" method="post">
+
+          <input type="email" id="email" name="email" value="" placeholder="email">
+          <input id="submit" type="submit" >
+        </form>
+
+    </header>
+    <section id="features">Features</section>
+      <section id="how-it-works">How</section>
+        <section id="pricing">Pricing</section>
```
Comment (danielamoreno699) [inline index.html:32]: I think you can reframe the semantic  . I noticed  you are using the tag to only display text, instead of displaying general content such as  t divs, h2, or p tags for example.

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+    flex-wrap: wrap;
+  }
+}
+
+.logo {
+  width: 60vw;
+}
+
+@media (max-width: 650px) {
+  .logo {
```
Comment (danielamoreno699) [inline styles.css:75]: I think you can group all the media queries you have in your code and place them at the end  of the css file in order to improve code readability.

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,310 @@
...
+  text-transform: uppercase;
+  border: 3px black solid;
+  border-radius: 2px;
+  background: transparent;
+  cursor: pointer;
+}
+
+.grid {
+  display: flex;
+}
```
Comment (danielamoreno699) [inline styles.css:51]: I noticed you are including classes that are not related to your HTML code. Maybe you can  check what tags and classes you are using and  remove those  that are not used.

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,310 @@
...
+  }
+}
+
+nav > ul {
+  width: 35vw;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+}
+
```
Comment (danielamoreno699) [inline styles.css:124]: I suggest you to write first **nav > ul** and then **nav > li**, since **ul** is  higher in the hierarchy. In your html code, you grouped the **li** inside the **ul**, therefore **ul** goes first .

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,310 @@
...
+@media (max-width: 650px) {
+  nav > ul {
+    flex-direction: column;
```
Comment (danielamoreno699) [inline styles.css:127]: Again, I think you are using the media query in the wrong place

File: styles.css [hunk 6]
```diff
@@ -0,0 +1,310 @@
...
+  justify-content: center;
+  height: 125px;
+  width: 20vw;
+  color: darkorange;
+}
+
+@media (max-width: 550px) {
+  #features .icon {
+    display: none;
+  }
```
Comment (danielamoreno699) [inline styles.css:188]: Please, be careful where you place the media query in your code.

**Review by danielamoreno699 — COMMENTED** [review]
### Changes required ♻️ 

**please check my comments below and follow the suggested changes ** 

- Before any submission, I recommend you to verify if your work has any linter errors and fix them if required. Remember that Linters help you detect coding errors. 
- Maybe you can document your repository , so you can track better your changes .
- I suggest you to check again your  css  file and work directly with each element and class in devtools ,  so you can improve your styling. 
- Maybe you cam follow my recommendations in relation with  media queries and unused classes that are written below.

**Review by Shisui6 — CHANGES_REQUESTED** [review]
Hi @danielamoreno699 , 

# Requires Changes 🍀
You have done a great job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! 🫡

## Highlights
- Descriptive PR title and summary ✅
- Follows Github flow ✅
- Good commit messages ✅

## Required Changes ♻️

- Great job utilising githubs `review changes`  feature✨ However, some of your comments come across a bit too harsh and straight forward😅. Its always better to be polite and respectful when giving reviews so as not to put down the person who is receiving the review. For example, `"I think it would be better to put the h1 inside the div tag because [reason for change] 😉` sounds much more encouraging, don't you think?

- Please always remember to state the reason for the requested change. The reason may be obvious to you, but the receiver may not not always understand why👍  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

You can also consider the following:

- [Optional] It would be nice if you also added emojis and gifs to your review😉. This helps make the review more human friendly and adds a nice and colourful touch✨

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by amejid — APPROVED** [review]
Hi @danielamoreno699, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Good use of Github features :heavy_check_mark: 
- Comments are polite and state a reason :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
